### DISCUSSION TITLE
add a default adapter or adapter priority list

### DISCUSSION THREAD

**Discussion by the-dbp** [snappysnappydog/Simple-IP-Config#141] [Ideas]
whenever I open  simple ip config and the last used adapter is not available simple ip config selects my bluetooth adapter, and very often I mistakenly set the bluetooth adapter to one of my static ips which i can then no longer set to the correct adapter.

I cannot change the bluetooth adapter back to auto for some reason so I either have to set a new ip to the adapter or save a new static ip on my profile, which is usually faster.

it would be nice if I can say "select ethernet if available" and it would be even nicer if can manually drag my adapters in the order I would like simple ip config to open into when the last used adapter is not available.

**Comment by KurtisLiggett** [comment]
I'll think about this one for a bit.
Would it not be better, if the adapter is not available, just make it not select any adapter (blank)? That way, if you don't realize it changed it would just fail and not set the next available adapter.

I do admit I've done this so many times while testing the program. I don't know why I never thought to fix it. :smile:

**Reply by the-dbp** [reply]
That would work!

**Comment by KurtisLiggett** [comment]
Now available with release v2.9.7 - [Download Here](https://github.com/KurtisLiggett/Simple-IP-Config/releases/tag/2.9.7)
